Tests/LibWeb/Text/expected/Wasm/WebAssembly-instantiate.txt

```diff
@@ -1,16 +1,52 @@
 -------------
 ArrayBuffer
 -------------
-Hello from wasm!!!!!!
```

Comment:
I don't understand why this worked before, if we have always failed to do stuff with the imports?

Reply:
#fixed